src/solvers/coin_cbc.rs

```diff
@@ -137,7 +139,14 @@ impl SolverModel for CoinCbcProblem {
         let solution = self.model.solve();
         let raw = solution.raw();
         match raw.status() {
-            Status::Stopped => Err(ResolutionError::Other("Stopped")),
+            Status::Stopped => {
+                if raw.is_seconds_limit_reached() {
+                    let solution_vec = solution.raw().col_solution().into();
+                    Ok(CoinCbcSolution{ status: SolutionStatus::TimeLimit, solution, solution_vec })
+                } else {
+                    Err(ResolutionError::Other("Stopped"))
+                }
```

Comment:
We need to add a case for `GapLimit` here, but I need to come up with a different problem because CBC seems to solve knapsack optimally in the first iteration almost every single time